code/src/components/loader.css

```diff
@@ -0,0 +1,103 @@
+.loader {
+  bottom: 0;
+  left: 0;
```

Comment:
Those styles are not going to work without `position: absolute`

Reply:
True, remainings from trial and error. Removed now.